ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-evpn-mpls.yml

```diff
@@ -41,7 +41,7 @@ router_bgp:
       send:
         limit: 10
     next_hop_mpls_resolution_ribs:
-      - rib_type: "tunnel-rib-colored"
+      - rib_type: "tunnel-rib colored"
```

Comment:
I don't think we should change the input from the user. Let's just change the configs.

Reply:
Fixed.